### DISCUSSION TITLE
Raylib physac library bindings

### DISCUSSION THREAD

**Discussion by AlecTroemel** [janet-lang/janet#1007] [Show and tell]
Hey ya'll, just wanted to show the [raylib physac library bindings](https://github.com/AlecTroemel/janet-physac) I've been working on! The API is pretty small, so hopefully I'll work through it all in the next couple of days. 

My C is still pretty rusty, so any feedback is welcome :)

**Comment by sogaiu** [comment]
@AlecTroemel I guess you moved on to working on [bindings for Chipmunk2D](https://github.com/AlecTroemel/janet-chipmunk) -- how has that been?

**Reply by AlecTroemel** [reply]
@sogaiu haha ya.. I got a pretty strong "dont use physac" from the raylib discord :laughing: 

For chipmunk, I've got far enough for the `hello-chipmunk.janet` example to run, and started work on joins+constraints. There are some implementation details I'm still trying to figure out, I just created some issues describing all the ones I could think of. 

My goal is to have the chipmunk bindings in a somewhat usable state for the Ludum Dare 51 game jam, so you can expect more work soon. And of course any help/tips/PR's are always welcome!
